frontend/src/ui/index.tsx

```diff
@@ -46,6 +44,7 @@ export const LinkList: React.FC<LinkListProps> = ({ items, ...rest }) => (
                 "&:first-child > li:first-child > a": {
                     borderRadius: `${SIDE_BOX_BORDER_RADIUS}px ${SIDE_BOX_BORDER_RADIUS}px 0 0`,
                 },
+                "li:only-child > a": { backgroundColor: "blue", borderRadius: 8 },
```

Comment:
I think you meant to remove the `blue` again :P

Reply:
Whoops... yeah I meant to remove that line